src/odemis/gui/comp/overlay/world_select.py

```diff
@@ -239,3 +242,268 @@ def on_motion(self, evt):
             WorldOverlay.on_motion(self, evt)
 
     # END Event Handlers
+
+
+class RectanglePointsSelectOverlay(WorldOverlay, RectangleMixin):
+
+    def __init__(self, cnvs, colour=gui.SELECTION_COLOUR, center=(0, 0)):
+        WorldOverlay.__init__(self, cnvs)
+        RectangleMixin.__init__(self, colour, center)
+
+        self.p_point1 = None
+        self.p_point2 = None
+        self.p_point3 = None
+        self.p_point4 = None
+
+        self._side1_label = self.add_label("", colour=(0.8, 0.8, 0.8), align=wx.ALIGN_RIGHT)
+        self._side2_label = self.add_label("", colour=(0.8, 0.8, 0.8), align=wx.ALIGN_RIGHT)
```

Comment:
Please explain what side 1 & 2 refer to. (typically a rectangle has 4 sides, so it's a bit confusing).
It'd be also useful to indicate when these labels are used. They are created empty.